README.md

```diff
@@ -13,6 +13,8 @@
 
 4. Run. `node index.js`
 
+5. Run. `node frames.js`
```

Comment:
this should be called as part of the main process. preferably add it after [here](https://github.com/WFCD/warframe-wikia-scrapers/pull/16/files/fb6a3a31cf2d4f80a2479406b484baf3a5b336de#diff-168726dbe96b3ce427e7fedce31bb0bcR121)